src/Tangential_complex/include/gudhi/Tangential_complex.h

```diff
@@ -624,6 +625,12 @@ class Tangential_complex {
 
     int max_dim = -1;
 
+    // Vertices to be inserted first by the create_complex method to avoid quadratic complexity.
+    // It isn't just [0, n) if some points have multiplicity (only one copy appears in the complex).
+    std::vector<typename Simplex_tree_::Vertex_handle> vertices(m_points.size());
+    std::iota(vertices.begin(), vertices.end(), 0);
```

Comment:
The comment does not match the code at all.

Reply:
Bad copy/paste of a comment. Fixed on [6b8da55](https://github.com/GUDHI/gudhi-devel/pull/762/commits/6b8da55930c5a76aefc6725d3cffb17bab5438d0)